src/main/java/uk/gov/hmcts/reform/professionalapi/util/RefDataUtil.java

```diff
@@ -47,16 +48,15 @@
 import uk.gov.hmcts.reform.professionalapi.domain.SuperUser;
 import uk.gov.hmcts.reform.professionalapi.domain.UserAccountMap;
 
+@Component
 @Slf4j
 public class RefDataUtil {
 
     private RefDataUtil() {}
 
-    @Value("${defaultPageSize}")
-    public static final int DEFAULTPAGESIZE = 10;
+    public static int defaultPageSize;

```

Comment:
make private instead of public

Reply:
Addressed in this commit: https://github.com/hmcts/rd-professional-api/pull/649/commits/7c8443e1fc31e49bd9979dbf419c336f5e5d034a